client/src/pages/Bookings/Bookings.tsx

```diff
@@ -0,0 +1,14 @@
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+import Navbar from '../../components/Navbar/Navbar';
+function Bookings(): JSX.Element {
+  return (
+    <Grid container>
+      <Navbar />
```

Comment:
What about rendering the Navbar outside the Switch component in the App.tsx to make it persist across the app? :)